omniport/core/session_auth/views/login.py

```diff
@@ -64,6 +69,10 @@ def post(self, request, *args, **kwargs):
             response_data = {
                 'errors': serializer.errors,
             }
+            session_auth_log(
+                f'Invalid login attempted with credentials {serializer.data}',
```

Comment:
Log message contains less information: Which `person` attempted an invalid login?

Reply:
Done: removed password from being printed on the log

Reply:
Please check the resolved changes